docs/src/man/reshaping_and_pivoting.md

```diff
@@ -72,14 +291,51 @@ This repeats the original columns N times where N is the number of columns stack
 
 For more details on the storage representation, see:
 
-```julia
+```jldoctest
+#FIXME
 dump(stackdf(iris))
 ```
 
 None of these reshaping functions perform any aggregation. To do aggregation, use the split-apply-combine functions in combination with reshaping. Here is an example:
 
-```julia
-d = stack(iris)
-x = by(d, [:variable, :Species], df -> DataFrame(vsum = mean(Nulls.skip(df[:value]))))
-unstack(x, :Species, :vsum)
+```jldoctest
+julia> using DataFrames, CSV
+
+julia> iris = CSV.read(joinpath(Pkg.dir("DataFrames"), "test/data/iris.csv"));
+
+julia> d = melt(iris, :Species);
+
+julia> head(d)
+6×3 DataFrames.DataFrame
+│ Row │ variable    │ value │ Species │
+├─────┼─────────────┼───────┼─────────┤
+│ 1   │ SepalLength │ 5.1   │ setosa  │
+│ 2   │ SepalLength │ 4.9   │ setosa  │
+│ 3   │ SepalLength │ 4.7   │ setosa  │
+│ 4   │ SepalLength │ 4.6   │ setosa  │
+│ 5   │ SepalLength │ 5.0   │ setosa  │
+│ 6   │ SepalLength │ 5.4   │ setosa  │
+
+julia> x = by(d, [:variable, :Species], df -> DataFrame(vsum = mean(parse.(Float64, df[:value]))));
```

Comment:
Add a line break. To work around the CSV.jl parsing bug, probably better go back to latest version for now.